### ISSUE TITLE
Feature request: Installed Content Window should be on one simple page

### ISSUE THREAD

**Issue by CuddleBear92** [xenia-manager/xenia-manager#81]
So the current Installed Content manager is all split in 7 different tabs that is selected with an slow and clunky drop-down.
To fix this, having a simple list on a single tab/page would make more sense with columns for the type of entry it is, the size it uses or other stats. Think similar to the Cemu Title Manager where everything is in a single list with and easy filter, multiple columns and all sortable.

![image](https://github.com/user-attachments/assets/e385dadc-8dc7-476a-baf8-8bc42a6a2adb)

**Comment by shazzaam7** [comment]
This was my original plan to have everything looking like this, but for 1 game instead of all of the stuff into 1 window like in your screenshot. As I was trying to make it work I noticed that this type of control in C# WPF isn't really that good so I just decided to do a simple ComboBox that is used to switch between different content and then show everything in a list.

Later on I decided to move the importing/exporting of save files into this window, so the ComboBox/ListBox solution fit even better.
Right now I'll leave it like this, but maybe in a future when all of the features are done and all I need to do is maintain it, I can revisit the UI and change it.
